src/NzbDrone.Common.Test/ConfigFileProviderTest.cs

```diff
@@ -141,8 +144,30 @@ public void SaveDictionary_should_save_proper_value()
 
             Subject.SaveConfigDictionary(dic);
 
+            Subject.Port.Should().Be(port);
+        }
+
+        [Test]
+        public void SaveDictionary_should_only_save_specified_values()
+        {
+            int port = 20555;
+            int origSslPort = 20551;
+            int sslPort = 20552;
+
+            WithPartialDiskProvider();
```

Comment:
A bit torn about this, I could also just setup the mock to return the correct Config xml code.